org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/GradleDistributions.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2018 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.buildship.core;
+
+import java.io.File;
+import java.net.URI;
+
+/**
+ * Contains factory methods for all supported Gradle distribution types.
+ *
+ * @author Donat Csikos
+ * @since 3.0
+ * @noimplement this interface is not intended to be implemented by clients
+ */
+public abstract class GradleDistributions {
```

Comment:
❌ Not needed, can be part of `GradleDistribution`

Reply:
Fixed.